cli/command/service_test.go

```diff
@@ -86,49 +72,35 @@ func TestDeleteService(t *testing.T) {
 		DeleteService("svc_id").
 		Return(nil)
 
-	c := testutils.NewTestContext(t, []string{"svc_name"}, nil)
-	if err := command.delete(c); err != nil {
+	if err := testutils.RunApp(command, "l0 service delete svc_name"); err != nil {
```

Comment:
Should we do all inputs, for consistency? Makes it easier to read in my opinion.
```
input := "l0 service delete svc_name"
```

Reply:
It works for this one but if the commands are really long I bet wrapping the line will be pretty weird looking.... 

I kind of like the other concatenated `input`s, it reminds me of something you'd do after programming in Go for 3+ years 😉

Reply:
I agree. I meant more just pulling that line out and doing an input var for all commands. I wasn’t saying do away with the concatenation.